tests/shellcheck/ignores.sh

```diff
@@ -5,4 +5,5 @@ ignore_patterns=(
 dockerfile/
 scripts/
 widgets/
+webui-desktop  # This is just a temporary file, to be replaced with cockpit-desktop once 'unshare' is not filtered out by lorax
```

Comment:
Reasoning is wrong, unshare is now present
https://github.com/weldr/lorax/pull/1222

Reply:
Updated the reasoning.